ampd/src/handlers/solana_verify_worker_set.rs

```diff
@@ -208,21 +217,7 @@ where
             }
         };
 
-        let account_data = match self.rpc_client.get_account_data(&pub_key).await {
```

Comment:
seems (if [this](https://github.com/eigerco/axelar-amplifier/pull/16/commits/f085785542045aee51d88fdf5e074e28a963a463#r1585634876) is correct) we already have all needed information. So we can remove the RPC call.